WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40346,13 +40365,22 @@ e_latin_empire = {
 	capital = 496 # Byzantion
 	
 	short_name = yes
+	dynasty_title_names = no
 	
 	
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
```

Comment:
First of all the two are partially redundant, secondly it makes very little sense that Greeks would form this title and thirdly I don't know why you added this at all.